workspaces/agent-cli/src/commands/run.ts

```diff
@@ -1,25 +1,30 @@
 import { Command, flags } from '@oclif/command';
-import { SaasCaptureSaver } from '@useoptic/cli-shared';
+import {
+  fromOptic,
+  ICaptureSaver,
+  SaasCaptureSaver,
+} from '@useoptic/cli-shared';
 import { Config } from '../config';
 //@ts-ignore
 import jwtDecode from 'jwt-decode';
+//@ts-ignore
+import niceTry from 'nice-try';
```

Comment:
🚨 swallowing errors, pretty much never the thing we want to do. I totally agree that the UX should be nice and that Optic service being down shouldn't stop services. Swallowing errors, however, is the total enemy to visibility and complete ignores that thrown errors could have put the process in an undefined state. 

I don't see it used anywhere, but it shouldn't even be imported. We have to do the work of actually hardening this, not just hide the issues!